modules/microshift-lvms-system-requirements.adoc

```diff
@@ -11,7 +11,9 @@ Using LVMS in {product-title} requires the following system specifications.
 [id="lvms-volume-group-name_{context}"]
 == Volume Group Name
 
-The default integration of LVMS uses a volume group named `rhel`. You can change the default name of the volume group in the configuration file. For details, read the "Configuring the LVMS" section of this document.
+The default integration of LVMS selects the default volume group dynamically. If there are no volume groups on the {product-title} host, LVMS is disabled. If there is only one volume group on the {product-title} host, that volume group is used. If there are multiple volume groups,  the group `microshift` is used. If the `microshift` group is not found, LVMS is disabled. If you want to use a specific volume group, LVMS must be configured to select that volume group.
```

Comment:
```suggestion
The default integration of LVMS selects the default volume group dynamically. If there are no volume groups on the {product-title} host, LVMS is disabled. If there is only one volume group on the {product-title} host, that volume group is used. 

If there are multiple volume groups,  the group `microshift` is used. If the `microshift` group is not found, LVMS is disabled. If you want to use a specific volume group, LVMS must be configured to select that volume group.
```

Reply:
All of the if statements were a bit confusing when I read them. I think breaking it up into chunks might help. I also wondered if, `If there is only one volume group on the {product-title} host, that volume group is used.` If not, I _think_ the first two parts are easier to put back together because they explain how LVMS is disabled.

Reply:
Those changes are clearer. +1